test/k8sT/Services.go

```diff
@@ -2909,9 +2909,16 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 		// Run on net-next and 4.19 but not on old versions, because of
 		// LRU requirement.
 		SkipItIf(func() bool {
-			return helpers.DoesNotRunOn419OrLaterKernel() || helpers.RunsOnGKE()
+			return helpers.DoesNotRunOn419OrLaterKernel()
 		}, "Supports IPv4 fragments", func() {
-			DeployCiliumAndDNS(kubectl, ciliumFilename)
+			options := map[string]string{}
+			// On GKE we need to disable endpoint routes as fragment tracking
```

Comment:
Why don't we deploy Cilium with the following options instead?
	```
options := map[string]string{
				"endpointRoutes.enabled": "false",
			}
```

Reply:
Good question. That's what I tried first, the first time I disabled endpoint routes on GKE, but it just gets overwritten by `gke.enabled`. So we have to instead disable `gke.enable` to let `endpointRoutes.enabled` take its default value (false).

Reply:
Thanks, it might be worth mentioning this in the code.